src/components/common/Button/Button.module.scss

```diff
@@ -4,6 +4,7 @@
 // }
 
 .main {
+  @extend %hover-effect;
```

Comment:
Rozszerzenie dodajemy do selekotra `:hover` i to się tyczy wszystkich przypadków

Reply:
Dlaczego do selektora, a nie do rodzica? Przecież transition ustawiamy na rodzicu.

Reply:
Tak Aniu jak najbardzije masz rację i przepraszam za wprowadzenie w błąd. Następnym razem poczekaj na dopowiedź zanim wprowadzisz zmiany, bo to nie jest tak, że nikt nie jest nieomylny , albo czasami się rozpędzi...

Reply:
Nie ma problemu